templates/todo/projects/csharp-sql-swa-func/.repo/bicep/azure.yaml

```diff
@@ -10,7 +10,7 @@ services:
     language: js
     host: staticwebapp
     hooks:
-      postpackage:
+      predeploy:
         posix:
           shell: sh
           run: node entrypoint.js -o ./build/env-config.js
```

Comment:
Consider wrapping this code into an NPM script managed by the template.  Something like `azdpredeploy`.  Then our hook can just run `npm run azdpredeploy` ?

Reply:
I think the users would think about "creating configuration for my pacakged application" as a decoupled feature from `azd`. Introducing azd-specific artifacts in `src` seems like a weird thing to do?

Reply:
Just a suggestion. I don't have strong feelings either way.

Reply:
Yep, no worries. I think making the ToDo templates slimmer may be a good thing to do for us. Let's wait for user feedback before deciding one way or another.